test/common.js

```diff
@@ -2,6 +2,7 @@
 
 global.chai = require('chai').use(require('sinon-chai'));
 global.should = chai.should();
+global.expect = chai.expect;
```

Comment:
ah i didn't know that, I don't think HMPO do either as they use expect when expecting an undefined value